src/main/java/com/liveramp/captain/daemon/CaptainJoblet.java

```diff
@@ -188,6 +189,11 @@ private void submitRequest(CaptainRequestConfig config) throws DaemonException {
       notifier.notify(subject, e, CaptainNotifier.NotificationLevel.ERROR);
       requestUpdater.cancel(id);
       return;
+    } catch (CaptainSubmissionException ce) {
+      String subject = String.format("%s: handle transient submission failed for request %s", CaptainAlertHelpers.getHostName(), id);
+      notifier.notify(subject, ce, CaptainNotifier.NotificationLevel.ERROR);
+      requestUpdater.ready(id);
```

Comment:
Why not just retry? Doesn't seem like such a bad thing if this counts towards the "attempt" counter

Reply:
Agreed with this -- seems like a good use case for retrying? Another way to think about it is that if this keeps failing then we don't want to retry silently forever probably

Reply:
Sure. I can mark it as retry.

Reply:
I thought the whole point of this was that we explicitly _do not_ want these to count towards the attempt counter and eventually get quarantined? Here's the ticket that describes the issue this change is attempting to solve: https://liveramp.atlassian.net/browse/DMBE-1480
The problem is that if we fail to submit 3 times, the request gets quarantined, letting future (potentially full-refresh) imports be processed before this one is.